Framework/PythonInterface/test/python/plugins/algorithms/AbinsBasicTest.py

```diff
@@ -200,6 +211,37 @@ def test_partial(self):
                                                    Tolerance=self._tolerance)
             self.assertEqual(result, True)
 
+    def test_numbered(self):
+        # Can request workspaces by atom number; these should add up to the usual elemental totals
```

Comment:
Is it possible to have a better name for this test, or a description in the comment?

Reply:
Done! Also removed a duplicate test definition that crept in at some point.